controllers/common.go

```diff
@@ -240,7 +241,11 @@ func (r *Reconciler) ReleaseLock(ctx context.Context, instance client.Object) bo
 		},
 	}
 
-	r.Client.Delete(ctx, &cm)
+	err := r.Client.Delete(ctx, &cm)
+	if err != nil && k8s_errors.IsNotFound(err) {
+		return false
+	}
```

Comment:
if there is an error during delete, which is not IsNotFound, is it correct that we return true?

Reply:
This should be fixed, thank you!:)